.changeset/breezy-impalas-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+'@tokens-studio/sd-transforms': patch
+---
+
+bug issue 267 fixed
```

Comment:
Linking the issue is good but also describe the change itself, a summary of what was fixed

Reply:
@florin01hyma you resolved the conversation but I don't see a change, did you push your changes?